option.go

```diff
@@ -273,7 +273,17 @@ func OptionDescription(description string) func(*BaseRoute) {
 // like the controller function name and the package name.
 func OptionAddDescription(description string) func(*BaseRoute) {
 	return func(r *BaseRoute) {
-		r.Operation.Description += "\n\n" + description
+		r.Operation.Description += description
+	}
+}
+
+// Description appends a description to the route.
+// By default, the description is set by Fuego with some info,
+// like the controller function name and the package name.
+func OptionOverrideDescription(description string) func(*BaseRoute) {
```

Comment:
```suggestion
// OptionOverrideDescription overrides the default description set by Fuego. 
// By default, the description is set by Fuego with some info, 
// like the controller function name and the package name.
func OptionOverrideDescription(description string) func(*BaseRoute) {
```

Reply:
If you don't want users  to use it from the root, you can then deprecate it. Here and suggest people to use the one in option package